kythe/web/site/examples.md

```diff
@@ -232,6 +232,53 @@ $ /opt/kythe/tools/runextractor cmake \
     ```
 
 
+## Extracting projects built with `make`
+
+Projects built with make can be extracted by substituting the C/C++ compiler with Kythe's cxx_extractor binary.
+
+Given a simple example project:
+
+```c++
+# main.cc
+
+#include <iostream>
+
+int main(int argc, char** argv) {
+    std::cout << "Hello" << std::endl;
+}
+```
+
+```shell
+# makefile
+
+all: bin
+
+bin: main.cc
+  $(CXX) main.cc -o bin
+
+```
+
+Extraction is done by setting the `CXX` environment variable as well as some environment variables that configure `cxx_extractor`.
```

Comment:
nit: setting the CXX *make* variable (make allows doing so via several mechanisms, including the environment).